src/main/java/io/kestra/plugin/ai/embeddings/Redis.java

```diff
@@ -0,0 +1,98 @@
+package io.kestra.plugin.ai.embeddings;
+
+import com.fasterxml.jackson.databind.annotation.JsonDeserialize;
+import dev.langchain4j.community.store.embedding.redis.RedisEmbeddingStore;
+import dev.langchain4j.data.segment.TextSegment;
+import dev.langchain4j.store.embedding.EmbeddingStore;
+import io.kestra.core.exceptions.IllegalVariableEvaluationException;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.property.Property;
+import io.kestra.core.runners.RunContext;
+import io.kestra.plugin.ai.domain.EmbeddingStoreProvider;
+import io.swagger.v3.oas.annotations.media.Schema;
+import jakarta.validation.constraints.NotNull;
+import lombok.Builder;
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import lombok.experimental.SuperBuilder;
+import redis.clients.jedis.DefaultJedisClientConfig;
+import redis.clients.jedis.HostAndPort;
+import redis.clients.jedis.JedisPooled;
+
+import java.io.IOException;
+
+@Getter
+@SuperBuilder
+@NoArgsConstructor
+@JsonDeserialize
+@Schema(
+    title = "Redis Embedding Store"
+)
+@Plugin(
+    beta = true,
+    examples = {
+        @Example(
+            full = true,
+            title = "Ingest documents into a Redis embedding store.",
+            code = """
+                id: document-ingestion
+                namespace: company.team
+
+                tasks:
+                  - id: ingest
+                    type: io.kestra.plugin.ai.rag.IngestDocument
+                    provider:
+                      type: io.kestra.plugin.ai.provider.GoogleGemini
+                      modelName: gemini-embedding-exp-03-07
+                      apiKey: my_api_key
+                    embeddings:
+                      type: io.kestra.plugin.ai.embeddings.Redis
+                      host: localhost
+                      port: 6379
+                      indexName: embeddings
+                    fromExternalURLs:
+                      - https://raw.githubusercontent.com/kestra-io/docs/refs/heads/main/content/blogs/release-0-22.md
+                """
+        )
+    }
+)
+public class Redis extends EmbeddingStoreProvider {
+
+    @NotNull
+    @Schema(title = "The database server host")
+    private Property<String> host;
+
+    @NotNull
+    @Schema(title = "The database server port")
+    private Property<Integer> port;
+
+    @Schema(title = "The index name", description = "Default value: \"embedding-index\".")
```

Comment:
```suggestion
    @Schema(title = "The index name")
```

Default value is automatically documented